Ublox gps

jneiva08 opened the pull request:
I have done footprints for most used gps modules from ublox.

MAX
datasheet: https://www.u-blox.com/sites/default/files/MAX-8-M8-FW3_HardwareIntegrationManual_%28UBX-15030059%29.pdf

![max](https://user-images.githubusercontent.com/8236353/42284640-0c6ba146-7fa5-11e8-899a-dd95937ef83c.png)

NEO
datasheet: https://www.u-blox.com/sites/default/files/NEO-8Q-NEO-M8-FW3_HardwareIntegrationManual_%28UBX-15029985%29_0.pdf

![neo](https://user-images.githubusercontent.com/8236353/42284635-098b837e-7fa5-11e8-8585-5d8e3e146949.png)

LEA
datasheet: https://www.u-blox.com/sites/default/files/LEA-M8S-M8T-FW3_HardwareIntegrationManual_%28UBX-15030060%29.pdf

![lea](https://user-images.githubusercontent.com/8236353/42284631-06716672-7fa5-11e8-995d-0200124d1183.png)

------------

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the footprint(s) you are contributing
- [x] An example screenshot image is very helpful 
- [ ] If there are matching symbol or 3D model pull requests, provide link(s) as appropriate
- [x] Check the output of the Travis automated check scripts - fix any errors as required

poeschlr commented:
Courtyard clearance should be 0.25mm. You have 0.5mm. (Unless there is a note somewhere in the datasheet or appnote that i missed.)

Pin 21 of _LEA is larger than all others. Is this on purpose?

![screenshot from 2018-07-27 14-39-37](https://user-images.githubusercontent.com/18327350/43321041-00ade4a0-91ab-11e8-9c15-3b06a34a46d5.png)

jneiva08 commented:
fixed on all footprints F.CrtYd to 0.25

lea:
![leo](https://user-images.githubusercontent.com/8236353/43419226-4250c028-9438-11e8-9245-4c821c33fb3a.png)
fixed pad 21.

max:
![max](https://user-images.githubusercontent.com/8236353/43419194-303530ea-9438-11e8-93f1-483f9bbb39a4.png)

neo:
![neo](https://user-images.githubusercontent.com/8236353/43419170-231bbdc0-9438-11e8-9f70-2e480145e490.png)

poeschlr commented:
Thanks
